apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/domain/Image.java

```diff
@@ -85,6 +86,7 @@ public static Status fromValue(String v) {
       protected int minDisk;
       protected int minRam;
       protected Resource server;
+      protected List<BlockDeviceMapping> blockDeviceMapping;
```

Comment:
This should be initialized with `ImmutableList.of()` (unless there's a really good reason not to). All of the other Collection types in the domain package are initialized this way. `blockDeviceMapping` should be the same.